catalog/acm/README.md

```diff
@@ -1,38 +1,43 @@
-config-mgmt-iam.yaml   iam.cnrm.cloud.google.com/v1beta1           IAMPartialPolicy         source-reader-sync-cluster-name-project-id  platform-namespace
-config-mgmt-iam.yaml   iam.cnrm.cloud.google.com/v1beta1           IAMServiceAccount        sa-acm-gke-cluster                          platform-namespace
-feat-config-mgmt.yaml  gkehub.cnrm.cloud.google.com/v1beta1        GKEHubFeature            feature-name                                platform-namespace
-membership.yaml        gkehub.cnrm.cloud.google.com/v1beta1        GKEHubMembership         membership-name                             platform-namespace
-services.yaml          serviceusage.cnrm.cloud.google.com/v1beta1  Service                  project-id-cluster-name-acm                 platform-namespace
-services.yaml          serviceusage.cnrm.cloud.google.com/v1beta1  Service                  project-id-cluster-name-gkehub              platform-namespace
-```
+|          File           |                 APIVersion                 |          Kind           |                      Name                       |   Namespace    |
+|-------------------------|--------------------------------------------|-------------------------|-------------------------------------------------|----------------|
+| acm-membership-api.yaml | serviceusage.cnrm.cloud.google.com/v1beta1 | Service                 | platform-project-id-cluster-cluster-name-gkehub | config-control |
+| acm-membership-api.yaml | serviceusage.cnrm.cloud.google.com/v1beta1 | Service                 | platform-project-id-cluster-cluster-name-acm    | config-control |
+| config-mgmt-csr.yaml    | gkehub.cnrm.cloud.google.com/v1beta1       | GKEHubFeatureMembership | acm-membership-cluster-name                     | config-control |
+| config-mgmt-iam.yaml    | iam.cnrm.cloud.google.com/v1beta1          | IAMServiceAccount       | sa-acm-cluster-name                             | config-control |
+| config-mgmt-iam.yaml    | iam.cnrm.cloud.google.com/v1beta1          | IAMPartialPolicy        | sa-acm-cluster-name                             | config-control |
+| config-mgmt-iam.yaml    | iam.cnrm.cloud.google.com/v1beta1          | IAMPartialPolicy        | source-reader-sync-cluster-name-project-id      | config-control |
+| feat-config-mgmt.yaml   | gkehub.cnrm.cloud.google.com/v1beta1       | GKEHubFeature           | feat-acm-cluster-name                           | config-control |
+| membership.yaml         | gkehub.cnrm.cloud.google.com/v1beta1       | GKEHubMembership        | hub-membership-cluster-name                     | config-control |
+| services.yaml           | serviceusage.cnrm.cloud.google.com/v1beta1 | Service                 | project-id-cluster-name-gkehub                  | config-control |
+| services.yaml           | serviceusage.cnrm.cloud.google.com/v1beta1 | Service                 | project-id-cluster-name-acm                     | config-control |
 
 ## Resource References
```

Comment:
At some point I'd like to move these into just being links in the table above.

Reply:
tracking in https://github.com/GoogleCloudPlatform/blueprints/issues/133